libtenzir/include/tenzir/concept/parseable/tenzir/data.hpp

```diff
@@ -71,11 +71,11 @@ struct data_parser : parser_base<data_parser> {
     // with named fields. Allowing a mixture of both would mean we'd
     // have to deal with ambiguous inputs.
     auto record_parser =
-        '<' >> ~as<record>(named_field % ',') >> trailing_comma >> '>'
+        '{' >> ~as<record>(named_field % ',') >> trailing_comma >> '}'
         // Creating a record with repeated field names technically violates
         // the consistency of the underlying stable_map. We live with that
         // until record is refactored into a proper type (FIXME).
-      | ('<' >> (unnamed_field % ',') >> trailing_comma >> '>')
+      | ('{' >> (unnamed_field % ',') >> trailing_comma >> '}')
```

Comment:
This should not be part of this PR; I think we need to discuss separately whether we want this now or wait for v5.0 to bundle this with the other planned language changes.